conans/client/conan_command_output.py

```diff
@@ -59,7 +59,7 @@ def date_handler(obj):
 
     def _read_dates(self, deps_graph):
         ret = {}
-        for ref, _ in sorted(deps_graph.nodes):
+        for ref, _, _ in sorted(deps_graph.nodes):
```

Comment:
A node now is:
`(reference, conan_file, original_reference)`
Why? it matters for `--update`. It is not the same to have a node with a reference `lib/1.0@conan/stable#34` with an original reference `lib/1.0@conan/stable#34` or with an original reference `lib/1.0@conan/stable`. In this later case you have to check for updates in the server, becase the user requested "the latest revision".

Could be stored only the revision as you suggested? Yes, it could be, but I think it is more direct and understandable this way.